src/crypto-providers/util.ts

```diff
@@ -171,13 +175,35 @@ const filterSigningFiles = (
   const poolColdSigningFiles = signingFiles.filter(
     (signingFile) => signingFile.type === HwSigningType.PoolCold,
   )
+  const mintSigningFiles = signingFiles.filter(
+    (signingFile) => signingFile.type === HwSigningType.Mint,
+  )
+  const multisigSigningFiles = signingFiles.filter(
+    (signingFile) => signingFile.type === HwSigningType.MultiSig,
+  )
   return {
     paymentSigningFiles,
     stakeSigningFiles,
     poolColdSigningFiles,
+    mintSigningFiles,
+    multisigSigningFiles,
   }
 }
 
+const hwSigningFileToPubKeyHash = (signingFile: HwSigningData): Uint8Array => (
+  Cardano.PublicKey.from_bytes(
```

Comment:
@davidmisiak I'd suggest to make an audit of which cardano libraries we import, and name them consistently. I'd prefer "cardanoSerialization" or something instead of making the distinction just on upper/lower case, and maybe our old library, which i'm not sure we are updating (?)... maybe a name reflecting "our/VL", "old", or both would be suitable (if we can't get rid of it --- does Adalite use it, or just hw-cli? or what does Nufi do?)

Reply:
Renamed in https://github.com/vacuumlabs/cardano-hw-cli/pull/98/commits/3dc8b1231f775650b1249ab607842d88df98067b - I suggest calling them `cardanoCrypto` and `cardanoSerialization`, so that there is no extra "mapping" between our imported name and the actual library name.

> does Adalite use it, or just hw-cli?

It seems to be actively used, see eg. https://github.com/vacuumlabs/adalite/pull/1195